distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -1,7 +1,6 @@
 ## Methods for operating on `Text` in Enso.
```

Comment:
Any reason why this file was not merged with `Text.enso`?

Not saying that it should, we were considering with @jdunkerley to even split this file into smaller ones as it is quite huge already.
Just asking because other `Extensions` files were mostly merged and this one is not so I'm wondering what is the rationale behind that.

Reply:
Can't remember the reason. Maybe I didn't want to do too much reshuffling. But I can give it a try!

Reply:
It's possible, but it makes the whole review even harder. I will keep it in the queue for the follow up PR